src/views/clusteroverview/utils/VirtctlPopup.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+
+import { ConsoleCLIDownloadModelGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import ExternalLink from '@kubevirt-utils/components/ExternalLink/ExternalLink';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
+
+import { VIRTCTL_DOWNLOADS } from './constants';
+import { K8sResourceKind } from './types';
+
+import './VirtctlPopup.scss';
+
+const VirtctlPopup: React.FC = () => {
+  const { t } = useKubevirtTranslation();
+  const [cliTools] = useK8sWatchResource({
+    groupVersionKind: ConsoleCLIDownloadModelGroupVersionKind,
+    isList: true,
+    namespaced: false,
+  });
+
+  const tools = Array.isArray(cliTools) ? cliTools : [...[cliTools]];
+  // skipcq: JS-0349
+  const virtctlObj = tools?.find(
+    (tool) => (tool as K8sResourceCommon)?.metadata?.name === VIRTCTL_DOWNLOADS,
+  ) as K8sResourceKind;
+
+  const displayName = virtctlObj?.spec?.displayName;
+  const links = virtctlObj?.spec?.links;
```

Comment:
im guessing you can deconstruct displayName and link in same line
const {displayName , links } = virtctlObj?.spec || {}

Reply:
Good call. Done!